util/reggen/gen_dv.py

```diff
@@ -72,6 +76,64 @@ def gen_core_file(outdir: str,
         yaml.dump(core_data, core_file, encoding='utf-8')
 
 
+def get_all_dv_base_names(dv_base_prefix: str) -> Dict[str, Dict[str, str]]:
```

Comment:
I had made this comment which seems to have vanished because you made amends to the commit. Adding it back here:
https://github.com/lowRISC/opentitan/pull/9007#discussion_r740731398